srml/contracts/src/tests.rs

```diff
@@ -1116,12 +1117,20 @@ fn restoration(test_different_storage: bool, test_restore_to_with_dirty_storage:
 		vec![acl_key, acl_key],
 	))));
 
-	let literal = "0105020000000000000021d6b1d59aa6038fcad632488e9026893a1bbb48581774c771b8f243206\
-		97f053200000000000000080100000000000000000000000000000000000000000000000000000000000000010\
-		0000000000000000000000000000000000000000000000000000000000000";
-
-	assert_eq!(encoded, literal);
-	assert_eq!(115, hex::decode(literal).unwrap().len());
```

Comment:
this check is to help if len is changing and also to ensure that the length of the buffer set in the code_restoration wat is correct `(i32.const 115)`

Reply:
Aaaaah, I see, thanks! Will write it down!